pkg/compare/compare_test.go

```diff
@@ -939,3 +940,137 @@ func jsonEqual(got, want interface{}) bool {
 	diff, _ := jsondiff.Compare(a, b, &jsondiff.Options{})
 	return diff == jsondiff.FullMatch
 }
+
+func BenchmarkCompareNestedArrayOperator(t *testing.B) {
```

Comment:
@jirevwe Kinda of a nitpick here, but this should be `b *testing.B` instead.

Reply:
Ah yes.